pkg/ccl/changefeedccl/metrics.go

```diff
@@ -320,6 +320,12 @@ var (
 		Measurement: "Errors",
 		Unit:        metric.Unit_COUNT,
 	}
+	metaChangefeedPauses = metric.Metadata{
+		Name:        "changefeed.pauses",
```

Comment:
perhaps num_paused is a better name?

Reply:
I ended up putting the metric in `jobs/metrics.go`. I went with `currently_paused` to match the preexisting `currently_idle` metric.